README.md

```diff
@@ -1,3 +1,9 @@
 # 3d-app-desktop
 
-Workflow tutorial: [here](https://docs.google.com/document/d/1QJ_aPE3kYIXs6J7uHMTFpVwkObSrH2yibf9JSNLUqNc/edit?usp=sharing)
+### To run the project locally:
+
+- Download the Java JDK: `https://www.oracle.com/java/technologies/downloads/` 
+- Download the plugin `Lombok` in Intelij Idea for the development environment to recognize the syntax (it does not affect the operation of the program)
+
```

Comment:
Do I need maven or gradle?

Reply:
Actually I don't understand, what means your question, because our project uses Maven, but install it or not? I didn't it but the project runs.

Reply:
Then you are using Maven which IntellijIdea provides. However, if a person who is trying to run the project uses VS Code, for example, they will need to install Maven manually. So it's better to add it to the requirements